src/content/whats-new/2024/02/whats-new-02-16-infra-otel-guids.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Changes to host GUIDs for hosts instrumented with OpenTelemetry"
+summary: "If you've instrumented infrastructure hosts with OpenTelemetry, you will need to update your host GUIDs to avoid breaking changes to your alerts and dashboards"
+releaseDate: '2024-02-16'
+learnMoreLink: 'https://docs.newrelic.com/docs/docs/new-relic-solutions/new-relic-one/core-concepts/what-entiy-new-relic'
+---
+
+On March 27, 2024, the `entity.guid` value for hosts captured by the OpenTelemetry collector will change. This change only affects customers who use OpenTelemetry to instrument their infrastructure hosts. 
+
+## What's changing?
+
+* This change applies to customers who use the OpenTelemetry collector for instrumenting hosts. This change doesn't affect other hosts instrumented with our other agents. 
+* On March 27, 2024, host IDs for hosts instrumented with OpenTelemetry will stop working after 24 hours and new host entities will appear for the same resources but with new GUIDs.
```

Comment:
I think this is better explained in the public post: https://forum.newrelic.com/s/hubtopic/aAXPh0000000hGH/host-entities-from-otel-agent-to-use-new-guid
>This change is applicable for customers who use OTel agent for instrumenting Hosts. With this change, if you have alerts or custom dashboards created using hardcoded GUIDs in them, it will not work going forward. 
Dashboards and alerts might be impacted only if they hardcoded the entity.guid value for OTel hosts.